sass/compat/themes/buddyboss-theme/_woocommerce-form-fields-select2.scss

```diff
@@ -0,0 +1,23 @@
+// ------------------------------------
+// SELECT2 FORM FIELDS
+// ------------------------------------
+
+body#{ $_body-theme-selector }#{ $_body-page-selector } .select2-container .select2-dropdown {
+	max-width: none;
+    border: 1px solid var(--bb-content-border-color);
+    box-shadow: none;
+}
+
+body#{ $_body-theme-selector }#{ $_body-page-selector } .select2-container .select2-dropdown--below {
+    border-top-left-radius: 0;
+    border-top-right-radius: 0;
+}
+
+body#{ $_body-theme-selector }#{ $_body-page-selector } .select2-container .select2-dropdown--above {
+    border-bottom-left-radius: 0;
+    border-bottom-right-radius: 0;
```

Comment:
Are these theme specific issues? Have you tried to fix this with CSS variables:

```scss
// ------------------------------------
// DEFINE CSS VARIABLES
// ------------------------------------
:root {
	--fluidcheckout--field--height: #{ remsize( 4.4rem ) };
	--fluidcheckout--field--border-radius: #{ remsize( .3rem ) };
}
```
Example from `sass/compat/themes/aora/__imports.scss`

Reply:
This is a `select2` dropdown-specific issue. The theme overrides the default styles for the dropdowns for which from what I understand FC doesn't have variables/styles. I've added comments to indicate that, but please let me know if there's a misunderstanding.